plugins/pulp_rpm/plugins/importers/yum/pulp_solv.py

```diff
@@ -676,20 +681,26 @@ def get_repo_units(self, repo_id):
         )
 
     def get_units_from_solvables(self, solvables):
-        """Map a whole list of solvables back into their Pulp units.
+        """Map a whole list of solvables back into their Pulp units, keyed by the repo they
```

Comment:
What I did *not* do - change the function of ```associate.py::copy_rpms()``` to work with this new output, because that will require more consideration.